src/pages/Introduce/Introduce.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import "./Introduce.scss";
+import { Link } from "react-router-dom";
+import { useNavigate } from "react-router-dom";
+
+const Introduce = () => {
+  const navigate = useNavigate();
+
+  const handleInput = () => {
+    console.log("input 태그 동작");
+  };
+
+  const handleButton = () => {
+    navigate("/product");
+  };
+
+  return (
+    <div className="contentContainer">
```

Comment:
```suggestion
    <div className="introduce">
```
가장 최상위 부모태그는 컴포넌트의 이름과 동일하게 구현해주세요
단, 소문자로 시작하는 형태로 구현해주세요!